pkg/actors/actors_test.go

```diff
@@ -790,6 +791,43 @@ func TestOverrideTimerCancelsMultipleActiveTimers(t *testing.T) {
 	})
 }
 
+func timerRepeats(ctx context.Context, t *testing.T, repeats uint32) {
+	requestC := make(chan testRequest, 10)
+	appChannel := mockAppChannel{
+		requestC: requestC,
+	}
+	testActorsRuntime := newTestActorsRuntimeWithMock(&appChannel)
+	actorType, actorID := getTestActorTypeAndID()
+	fakeCallAndActivateActor(testActorsRuntime, actorType, actorID)
+
+	timer := createTimerData(actorID, actorType, "timer", "2s", "2s", repeats, "callback", "data")
+	err := testActorsRuntime.CreateTimer(ctx, &timer)
+	assert.Nil(t, err)
+
+	var cnt uint32
+LOOP:
+	for {
+		select {
+		case request := <-requestC:
+			assert.Equal(t, timer.Data, request.Data)
+			cnt++
+		case <-time.After(15 * time.Second):
+			assert.Equal(t, repeats, cnt)
```

Comment:
This hard code time(15s) may cause fail with a bigger `repeats`, like 10.

Reply:
thanks, addressed!